src/modules/admins/admin-operations.resolver.ts

```diff
@@ -93,4 +95,29 @@ export class AdminOperationsResolver {
       throw new ApolloError(error);
     }
   }
+
+  @Mutation(() => Boolean)
+  @UseGuards(GqlAdminAuthGuard)
```

Comment:
could you extract in a args object all this parameters

Reply:
Done